tower-balance/examples/demo.rs

```diff
@@ -125,7 +119,7 @@ fn main() {
     rt.shutdown_on_idle().wait().unwrap();
 }
 
-type Error = Box<::std::error::Error + Send + Sync>;
+type Error = Box<dyn ::std::error::Error + Send + Sync>;
```

Comment:
I think here we can `Box<dyn std::error::Error + Send + Sync>`

Reply:
Good point, fixed.